javaslang-benchmark/src/test/java/javaslang/collection/ArrayBenchmark.java

```diff
@@ -7,6 +7,7 @@
 import java.util.*;
 
 import static javaslang.JmhRunner.getRandomValues;
+import static javaslang.collection.Collections.areEqual;
```

Comment:
I'm totally fine with this commit! Great that you separated the renaming into one commit!